website/description_issues.qmd

```diff
@@ -106,3 +106,29 @@ It is also not recommended to [mention package names](https://cran.r-project.org
 
 ------------------------------------------------------------------------
 
+# Using Authors@R
+
+
+## Problem
+
+You have not properly declared the authors, maintainers and copyright holders in the Authors@R section of the DESCRIPTION file.
+
+## Solution
+
+Add the `Authors@R` field in the DESCRIPTION file. 
+
+
+### Details
+
+You can find more roles in the help documentation by running `?utils::person`. Make sure the provided email address is actively monitored, as it will be the primary point of contact with CRAN for future updates and fixes.
+
+```{r}
+Authors@R: person("Jane", "Smith", email = "chef@cran-cookbook.com",
+  role = c("aut", "cre"))
+```
+
+
+::: callout-note
+If your package uses the older method of specifying separate `Maintainer` and `Author` fields in the DESCRIPTION file, it won’t be rejected solely because of this. However, this approach is less preferred.
```

Comment:
```suggestion
If your package uses the older method of specifying separate `Maintainer` and `Author` fields in the DESCRIPTION file, it will be rejected because of this.
```
There was a change a few months ago. Since R packages now have their own `doi`, using `Authors@R` is now mandatory. 
For old packages, declaring the Authors using the old system it is recommended to change. However, your package won't be taken off CRAN because of a missing `Authors@R`-field. 
We recommend changing soon, though (also said by Kurt Hornik in his keynote speech at useR!2024).